cmd/veto/tui_data.go

```
@@ -0,0 +1,148 @@
+		snapshot.Analytics = controlplane.AnalyticsSnapshot{
+			LocalCollection:       status.LocalCollection,
+			LocalPath:             status.LocalPath,
+			RetentionDays:         status.LocalRetentionDays,
+			RemoteSharing:         status.RemoteSharing,
+			RemoteTransportActive: status.RemoteTransportActive,
+		}
+	} else {
+		snapshot.Health = append(snapshot.Health, controlplane.HealthSnapshot{ID: "analytics.config", Status: "WARN", Message: "analytics preference could not be read"})
+	}
+
+	report := runDoctor(doctorOptions{offline: true}, defaultDoctorDeps())
```

golangci-lint (2.13.2)

[error] Function `runDoctor->checkDoctorReleaseIntegrity` should pass the context parameter

(contextcheck)